pr-data.csv

```diff
@@ -3392,6 +3402,13 @@ https://github.com/openpojo/openpojo,9badbcc4593e797accfed5e51bec9f2b843f0f67,.,
 https://github.com/openpojo/openpojo,9badbcc4593e797accfed5e51bec9f2b843f0f67,.,com.openpojo.validation.test.impl.SerializableTesterTest.shouldSkipOverNonSerializableClass,OD-Vic,Opened,https://github.com/OpenPojo/openpojo/pull/136,
 https://github.com/openpojo/openpojo,9badbcc4593e797accfed5e51bec9f2b843f0f67,.,com.openpojo.validation.test.impl.SetterTesterTest.shouldSuccessfullyValidateEvenIfLoggingFails,OD;UD,Opened,https://github.com/OpenPojo/openpojo/pull/136,
 https://github.com/openpojo/openpojo,9badbcc4593e797accfed5e51bec9f2b843f0f67,.,com.openpojo.validation.utils.ValidationHelperTest.shouldReportMissingASMProperly,OD-Vic,Opened,https://github.com/OpenPojo/openpojo/pull/136,
+https://github.com/OpenRefine/OpenRefine,3bff27b868a6a1b695f3855ae0f49e269c5ef7cd,main,com.google.refine.browsing.facets.ListFacetTests.serializeListFacet,ID,,,
+https://github.com/OpenRefine/OpenRefine,3bff27b868a6a1b695f3855ae0f49e269c5ef7cd,main,com.google.refine.browsing.facets.ListFacetTests.testSelectedEmptyChoice,ID,,,
+https://github.com/OpenRefine/OpenRefine,3bff27b868a6a1b695f3855ae0f49e269c5ef7cd,main,com.google.refine.clustering.binning.BinningClustererTests.testSerializeBinningClusterer,ID,Opened,https://github.com/OpenRefine/OpenRefine/pull/5426,
```

Comment:
You should follow up on this; it's been a week since the developer provided feedback.

Reply:
Developer asked not to make change in the main code as it may cause some performance issue and try to solve the flaky test by making changes in the test code. But it is not that straight forward to fix in the test code since the code which has the assert is part of a generic test code which is used by multiple tests, some which which contain bigger jsons. I am trying to find a fix which does not end up breaking other testcases